kikar_hamedina/core/sitemap.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.sitemaps import Sitemap
+from mks.models import Member, Party
+from django.conf import settings
+from datetime import datetime
+import pytz
+
+class MkSitemap(Sitemap):
+    changefreq = "weekly"
+    priority = 0.5
+
+    def items(self):
+        return Member.objects.filter(is_current=True);
+
+
+    def lastmod(self, obj):
+        if (obj.facebook_persona != None and
```

Comment:
In python instead of writing:

`if A != None`

you can (and should) write:

`if A` (or `if not A` for the opposite)